packages/react/src/global.ts

```diff
@@ -8,6 +8,7 @@ declare module 'focusable' {
   export = focusable;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
I don't understand the purpose of this ESLint rule. It's fine with namespaces in `.d.ts` files, but not in `.ts` or `.tsx` files. 🤷